src/Components/Telemetry.md

```diff
@@ -248,3 +248,30 @@ Aspire.StackExchange.Redis.OutputCaching:
 - Everything from `Aspire.StackExchange.Redis` plus:
 - Log categories:
   - "Microsoft.AspNetCore.OutputCaching.StackExchangeRedis"
+
+Aspire.Oracle.EntityFrameworkCore:
```

Comment:
(nit) Can you keep these are alphabetically ordered?

Reply:
Of course, I didn't understand the ordering, sorry.